### DISCUSSION TITLE
Lesson 2 BigNumber error

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#107] [Q&A] [answered]
Hello, i followed along, even tried copy pasting the code from here....and i continue to get this error

_call to StorageFactory.sfGet errored: Error encoding arguments: Error: invalid BigNumber string (argument="value", value="", code=INVALID_ARGUMENT, version=bignumber/5.4.1)_

I can successfully createSimpleStorage 
I can successfully do the sfStore with with 0, 22
but whenever i press sfGet, i for some reason get that error. I have looked over both files and they are identical to what has been posted on here. 

any thoughts?

**Comment by ghost** [comment, marked as answer]
nevermind, I shouldn't program when I'm tired this question can be closed now.

**Reply by PatrickAlphaC** [reply]
Ahaha nice work!

**Comment by ace-brown** [comment]
I've got the same error. Can anybody help me to solve it?

**Reply by ghost** [reply]
well, my error was just a oversight of my behalf. I wasn't putting the 0 in the respected place for sfget, i just kept pressing sfget and having that error returned.
